components/external_media_links/external_media_link.lua

```diff
@@ -0,0 +1,270 @@
+			:tag('span')
+				:css('font-style', 'italic')
+				:wikitext('(' .. args.note .. ')')
+	end
+
+	return display
+end
+
+function ExternalMediaLink._displayEvent(args)
+	local prefix = NON_BREAKING_SPACE .. 'at' .. NON_BREAKING_SPACE
+
+	if Logic.readBoolOrNil(args['event-link']) == false then
+		return prefix .. args.event
+	end
+
+	return prefix .. Page.makeInternalLink({}, args.event, args['event-link'])
+end
+
+function ExternalMediaLink._displayTranslation(args)
+	local translation = NON_BREAKING_SPACE .. '(trans. '
+		.. Flags.Icon({flag = args.translation, shouldLink = false})
+
+	if String.isEmpty(args.translator) then
+		return translation .. ')'
+	end
+
+	return translation .. NON_BREAKING_SPACE .. 'by' .. NON_BREAKING_SPACE .. args.translator .. ')'
+end
+
+function ExternalMediaLink.wrapper(args)
+	local wrapperInfoDisplay = mw.html.create('table')
+		:attr('border', 0)
+		:attr('cellpadding', 4)
+		:attr('cellspacing', 4)
+		:css('margin-bottom', '5px')
+
+	wrapperInfoDisplay
+		:tag('tr')
+			:tag('th'):wikitext('Author(s)'):done()
+			:tag('td'):wikitext(args.authors or ''):done():done()
+		:tag('tr')
+			:tag('th'):wikitext('Title'):done()
+			:tag('td'):wikitext(args.title or ''):done():done()
+		:tag('tr')
+			:tag('th'):wikitext('Date'):done()
+			:tag('td'):wikitext(args.date or ''):done():done()
+		:tag('tr')
+			:tag('th'):wikitext('URL'):done()
+			:tag('td'):wikitext(args.link or '')
+
+	local parsedArgs = {
+		date = args.date,
+		link = args.link,
+		title = args.title,
+		type = args.type and args.type:lower() or nil,
+		of = args.of,
+		event = args.event,
+		['event-link'] = args['event-link'] and mw.ext.TeamLiquidIntegration.resolve_redirect(args['event-link']) or nil,
+		language = args.language,
+		translation = args.translation,
+		translator = args.translator,
+	}
+
+	if args.authors then
+		local authors = Array.map(mw.text.split(args.authors, ','), ExternalMediaLink._cleanValue)
+		args.by_link1 = args.by_link1 or args.by_link
+		for authorIndex, author in pairs(authors) do
+			parsedArgs['by' .. authorIndex] = author
+			parsedArgs['by_link' .. authorIndex] = args['by_link' .. authorIndex]
+		end
+	end
+
+	if args.subjects then
+		local subjects = Array.map(mw.text.split(args.subjects, ','), ExternalMediaLink._cleanValue)
+		for subjectIndex, subject in pairs(subjects) do
+			parsedArgs['subject' .. subjectIndex] = subject
+		end
+	end
+
+	if args.subject_organizations then
+		local orgs = Array.map(mw.text.split(args.subject_organizations, ','), ExternalMediaLink._cleanValue)
+		for orgIndex, org in pairs(orgs) do
+			parsedArgs['subject_organization' .. orgIndex] = org
+		end
+	end
+
+	return mw.html.create()
+		:wikitext('[[Special:FormEdit/ExternalMediaLinks|Go back to the form]]<br>')
+		:node(ExternalMediaLink.run(parsedArgs))
+		:node(wrapperInfoDisplay)
+end
+
+function ExternalMediaLink._cleanValue(value)
+	value = mw.text.trim(value)
+
+	if String.isEmpty(value) then
+		return
```

Comment:
```suggestion
		return nil
```
Re: https://github.com/Liquipedia/Lua-Modules/pull/2520/files#r1097285754
Otherwise there would be an error with table.insert inside the Array.map